### PR TITLE
#6 Add readme

### LINKED ISSUE #6: Add readme

### PR DESCRIPTION
**Pull request by mickmcgrath13**
closes #6

### CODE REVIEW

**Review by LeoDiazL — APPROVED** [review]
Looks good.

File: README.md [hunk 1]
```diff
@@ -1 +1,106 @@
...
+      name: Deploy
+      uses: bitovi/github-actions-deploy-stackstorm@main
+      with:
+        aws_access_key_id: ${{ secrets.AWS_ACCESS_KEY_ID}}
+        aws_secret_access_key: ${{ secrets.AWS_SECRET_ACCESS_KEY}}
+        aws_default_region: us-east-1
+        st2_auth_username: ${{ secrets.ST2_AUTH_USERNAME}}
+        st2_auth_password: ${{ secrets.ST2_AUTH_PASSWORD}}
+```
+
```
Comment (armab) [inline README.md:40]: It's worth mentioning which resources (AWS, ST2) this GH Action will rely on and create at a high level if user specifies this example config.

Comment (mickmcgrath13) [reply README.md:40]: good point. added a brief overview

File: README.md [hunk 2]
```diff
@@ -1 +1,106 @@
...
+## Customizing
+
+### Inputs
+
+The following inputs can be used as `step.with` keys
+
+| Name             | Type    | Description                        |
+|------------------|---------|------------------------------------|
+| `checkout` | Bool |  Specifies if this action should checkout the code (Default is `true`) |
```
[the remark this answers is not in the document]
Comment (mickmcgrath13) [reply README.md:49]: This property tells the action whether or not to run `uses: actions/checkout@v3` before running BitOps.  It has nothing to do with ansible. I'll update the docs a bit

**Review by LeoDiazL — APPROVED** [review]

**Review by LeoDiazL — APPROVED** [review]

**Pull request merged**